src/client/src/orangehrmBuzzPlugin/components/UpcomingAnniversary.vue

```diff
@@ -0,0 +1,172 @@
+        <div
+          v-for="anniversary in isViewDetails
+            ? displayMoreAnniversaries
+            : anniversaries"
+          :key="anniversary"
+          class="orangehrm-buzz-anniversary"
+        >
+          <div class="orangehrm-buzz-anniversary-profile">
+            <div class="orangehrm-buzz-anniversary-profile-image">
+              <img
+                alt="profile picture"
+                class="employee-image"
+                :src="`../pim/viewPhoto/empNumber/${anniversary.empNumber}`"
+              />
+            </div>
+            <div class="orangehrm-buzz-anniversary-profile-details">
+              <oxd-text tag="p" class="orangehrm-buzz-anniversary-emp-name">
+                {{ anniversary.empName }}
+              </oxd-text>
+              <oxd-text tag="p" class="orangehrm-buzz-anniversary-job-details">
+                {{ anniversary.jobTitle }}
+              </oxd-text>
+            </div>
+          </div>
+          <div class="orangehrm-buzz-anniversary-duration">
+            <img
+              alt="year celebration"
+              class="orangehrm-buzz-anniversary-year-celebration"
+              :src="celebrationPic"
+            />
+            <div class="orangehrm-buzz-anniversary-durations-text">
+              <oxd-text
+                tag="p"
+                class="orangehrm-buzz-anniversary-duration-years"
+              >
+                {{ anniversary.anniversaryYear }} <br />
+                {{ $t('buzz.years') }}
+              </oxd-text>
+              <oxd-text
+                tag="p"
+                class="orangehrm-buzz-anniversary-duration-date"
+              >
+                {{ anniversary.joinedDate }}
+              </oxd-text>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div v-if="anniversariesCount > 5">
+        <oxd-text
+          tag="p"
+          class="orangehrm-buzz-anniversary-see-more"
+          @click="onSeeMore"
+        >
+          {{
+            isViewDetails ? $t('general.show_more') : $t('general.show_less')
+          }}
+        </oxd-text>
+      </div>
+    </oxd-sheet>
+  </div>
+</template>
+<script>
+import useLocale from '@/core/util/composable/useLocale';
+import Sheet from '@ohrm/oxd/core/components/Sheet/Sheet';
+import {APIService} from '@/core/util/services/api.service';
+import {parseDate, formatDate} from '@/core/util/helper/datefns';
+
+export default {
+  name: 'UpcomingAnniversaries',
+
+  components: {
+    'oxd-sheet': Sheet,
+  },
+
+  setup() {
+    const {locale} = useLocale();
+    const celebrationPic = `${window.appGlobal.baseUrl}/../dist/img/year_celebration.png`;
+
+    const http = new APIService(
+      window.appGlobal.baseUrl,
+      'api/v2/buzz/anniversaries',
+    );
+
+    return {
+      http,
+      locale,
+      celebrationPic,
+    };
+  },
+
+  data() {
+    return {
+      anniversaries: [],
+      displayMoreAnniversaries: [],
+      anniversariesCount: 0,
```

Comment:
anniversariesCount should also be a computed property.  derive it from `anniversaries.length`

Reply:
done